spark3/src/test/java/org/apache/iceberg/spark/source/TestSparkSchema3.java

```diff
@@ -1,23 +0,0 @@
-/*
```

Comment:
A potential complication: `SupportsCatalogOptions` doesn't allow for specifying the schema. I don't know how much people rely on this feature but it is a breaking change for the IcebergSource